docs/main.adoc

```diff
@@ -16,13 +15,21 @@ Versjon 1.0
 
 image::digitaliseringsdirektoratet.png[width=50%, pdfwidth=30vw]
 
-include::shared/download.adoc[]
+// include::shared/download.adoc[]
 
 *Status*: [yellow-background]#utgått# +
 *Publisert*: september 2017 +
 *Oppdatert*: 3. november 2020
 
-IMPORTANT: Denne spesifikasjonen er [yellow-background]#utgått#. Se https://data.norge.no/specification/dqv-ap-no/[DQV-AP-NO] som implementerer kravene i denne spesifikasjonen.
+// markere denne som "utgått" med fixed textbox her kodet direkte i html
+++++
+<div style="position: fixed; top: 40%; left: 50%; transform: translateX(-50%); background-color: #ffffff; border: 1px solid black; padding: 15px; box-shadow: 15px 15px 15px red; opacity: 0.85">
+  <p> <b>NB! Denne spesifikasjonen er utgått og vil ikke bli vedlikeholdt (inkl. ev. kjente feil/mangler). <br>
```

Comment:
`<br />`

og bruke `<strong>` i stedet for `<b>`

Reply:
OK

Reply:
dog ikke <br />